src/pages/_app.tsx

```diff
@@ -83,6 +83,9 @@ const MyApp: React.FC<MyAppProps> = ({ Component, pageProps }) => {
   }, [router, nextRoute, taxForm, taxFormUserInput])
 
   useEffect(() => {
+    const defaultFocus = document.querySelector(
+      '[data-focus]',
```

Comment:
Ja som za aby focus bol ak to pomoze accessibilite. Ale toto sa mi zda trochu ako take hackovanie, nebolo by lepsie dat`autoFocus` prop na inputy ktore to maju mat?

Reply:
Moj jediny problem s autofocus je ze sa nto proste zabudne :(
Ale chapem no ze totot tu sa uz zamotava :( Vyhodil som celu tu logiku a prehodim to neskor na autofocus